EasyPost.Tests/UtilitiesTests/AttributesTest.cs

```diff
@@ -18,29 +85,61 @@ public class AttributesTest
         public void TestGetMethodsWithAttributes()
         {
             // via type
-            IEnumerable<MethodInfo> methods = BaseCustomAttribute.GetMethodsWithAttribute<CrudOperations.Update>(typeof(Shipment));
+            IEnumerable<MethodInfo> methods = BaseCustomAttribute.GetMethodsWithAttribute<ExampleAttribute>(typeof(ExampleClass));
             Assert.True(methods.Any());
 
             // via instance
-            Shipment shipment = new();
-            methods = BaseCustomAttribute.GetMethodsWithAttribute<CrudOperations.Update>(shipment);
+            ExampleClass exampleClass = new();
+            methods = BaseCustomAttribute.GetMethodsWithAttribute<ExampleAttribute>(exampleClass);
             Assert.True(methods.Any());
         }
 
+        [Fact]
+        [Testing.Logic]
+        public void TestCanGetNonPublicAndStaticMethodsWithAttributes()
+        {
+            // via type
+            IEnumerable<MethodInfo> methods = BaseCustomAttribute.GetMethodsWithAttribute<ExampleAttribute>(typeof(ExampleClass));
+            // we should have 8 methods with the attribute
+            Assert.True(methods.Count() == 8);
+
+            // via instance
+            ExampleClass exampleClass = new();
+            methods = BaseCustomAttribute.GetMethodsWithAttribute<ExampleAttribute>(exampleClass);
+            // we should have 8 methods with the attribute
```

Comment:
Nit: The assertion says the same as this comment. This comment isn't helpful (in it's current form) and should either be removed or clarified as to **why** we expect 8